modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -169,23 +177,130 @@ Where the [n] is replaced by the value for each of the date and time elements th
 - `M` is the minute designator (e.g. `|PT1M|`)
 - `S` is the second designator (e.g. `|PT1S|`)
 
-.Transform
+`DatePeriod` is useful for operate date manipulation, such as addition or subtraction. +
+The following example shows how to subtract one year from a date using `DatePeriod`. The return type is a new `Date`:
```

Comment:
`substract`?

Reply:
Indeed `substract` is obsolete, it not used anymore.